### PR TITLE
Add reference to issue #47.

### PR DESCRIPTION
**Pull request by msporny**
Add reference to issue #47.

### CODE REVIEW

File: specs/paymentrequest.html [hunk 1]
```diff
@@ -249,6 +249,30 @@
...
+  }
+};
+
+// request payment and get a promise that will resolve to
+// a payment acknowledgement
+var payment = navigator.payment.request(request);
+        </pre>
+      </div>
+
       <p>The following example shows how to construct a <a><code>PaymentRequest</code></a> and begin the
```
Comment (adrianhopebailie) [inline specs/paymentrequest.html:252]: @msporny Can you reword this to be a lot simpler and leave the deeper explanation for users to read in the issue itself? I would also suggest placing this issue marker alongside those for #15 and #16 which also relate to the shape of the API.

Suggestion:

``` html
<div class="issue" data-number="47" title="Should a payment request be just data, or a 
programmable object?"> There is an open issue about whether the payment request should 
be a programmable object or should be just pure data that can be operated on by methods.</div>
```

**Comment by msporny** [comment]
@adrianhopebailie wrote:

> Can you reword this 

Done and moved to where you suggested.

**Comment by adrianba** [comment]
Merged as 814702c5f937ebf66f075071ae13f29b20a13535
